test/test_rpc.py

```diff
@@ -1384,3 +1384,14 @@ def method(self):
 
         with eventlet.Timeout(1):
             [reply.result() for reply in replies]
+
+
+def test_rpc_over_ssl(container_factory, runner_factory, rabbit_ssl_config):
+    container = container_factory(ExampleService, rabbit_ssl_config)
+    container.start()
+
+    with entrypoint_hook(container, 'echo') as echo:
```

Comment:
`entrypoint_hook` triggers the entrypoint without actually using whatever transport it's connected to, so this isn't using AMQP.

You need to generate a real AMQP message, for example with the standalone RPC proxy:

``` python
with ServiceRpcProxy("service", rabbit_config) as proxy:
    assert proxy.echo("foo") == "foo"
``` 

The proxy can connect using SSL or not. In fact, it might be nice to test both.

Reply:
Thanks for comment! My code doesn't work now on real messaging. Going to find why